packages/wordcount/src/defaultSettings.js

```diff
@@ -1,3 +1,32 @@
+/**
+ * @typedef {import("./index").WordCountStrategy} WordCountStrategy
+ */
```

Comment:
Thanks, @aduth, that's useful and tried it in 3126c5db43. One drawback for me is that VSCode doesn't pick up the descriptions of the imported types, but that may be something in my setup that I haven't figured out.

Reply:
> One drawback for me is that VSCode doesn't pick up the descriptions of the imported types

I could see that, since while in practice we use this as a sort of import / aliasing, in reality it's creating a brand new type which happens to inherit _its type_ from where it's imported. That may very well not include details like the description. I've also seen issues when using this pattern to "import" a [generic type](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#template), where the details of the generic are lost.

Unfortunately, I don't know there's any other option here, aside from inlining every `import` where it's used.